src/video/cocoa/cocoa_v.mm

```diff
@@ -395,35 +395,11 @@ void QZ_GameSizeChanged()
  */
 static CocoaSubdriver *QZ_CreateWindowSubdriver(int width, int height, int bpp)
 {
-#if defined(ENABLE_COCOA_QUARTZ) || defined(ENABLE_COCOA_QUICKDRAW)
-	CocoaSubdriver *ret;
-#endif
-
 #if defined(ENABLE_COCOA_QUARTZ) && (MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_4)
```

Comment:
Can't this one be simplified away too? ENABLE_COCOA_QUARTZ should always be defined anyway.

Reply:
Somebody might in theory want to build a headless dedicated server.